src/constants/ActionTypes.js

```diff
@@ -48,6 +42,7 @@ export const UPDATE_USER_PREFERRED_NAME = makeActionStrings(
 /* These events will be fired on websocket messages */
 export const REPLACE_QUESTIONS = 'REPLACE_QUESTIONS'
 export const UPDATE_QUEUES = 'UPDATE_QUEUES'
+export const UPDATE_COURSES = 'UPDATE_COURSES'
```

Comment:
Has live course updating been added via Websocket?

Reply:
lol oops no, I'll remove that